src/Menu/Menu.spec.js

```diff
@@ -109,4 +110,116 @@ describe('<Menu />', () => {
       assert.strictEqual(list.hasClass('test-class'), true, 'should have the user class');
     });
   });
+
+  describe('mount', () => {
```

Comment:
> I'm not sure I see the purpose of having separated tests?

Me neither, I wish we had @nathanhere point of view here. I have the feeling that it would be better moving everything under the `/src/COMPONENT` folder.